spec/unit/azure_server_create_spec.rb

```diff
@@ -95,6 +95,13 @@ def test_params(testxml, chef_config, role_name, host_name)
         expect {@server_instance.run}.to raise_error(SystemExit)
       end
 
+      it "raise error if neither provided --azure-dns-name nor --azure-vm-name" do
```

Comment:
it "raises an error if neither --azure-dns-name or --azure-vm-name are provided"